kpi/paginators.py

```diff
@@ -142,3 +142,10 @@ class TinyPaginated(PageNumberPagination):
     Same as Paginated with a small page size
     """
     page_size = 50
+
+
+class OrganizationPagination(PageNumberPagination):
```

Comment:
I would rename the class just to match what it is paginating (i.e.: organization members)